e2e/test/Helpers/logging/MsTestLogger.cs

```diff
@@ -25,10 +38,16 @@ internal MsTestLogger(TestContext testContext) : base()
             string operatingSystem = RuntimeInformation.OSDescription.Trim();
 
             // Add test related properties.
-            Properties.Add(LoggingPropertyNames.TestName, testContext.TestName);
-            Properties.Add(LoggingPropertyNames.TestClassName, testContext.FullyQualifiedTestClassName);
-            Properties.Add(LoggingPropertyNames.TargetFramework, targetFramework.FrameworkName);
-            Properties.Add(LoggingPropertyNames.OsInfo, operatingSystem);
+            MsTestProperties.Add(LoggingPropertyNames.TestName, testContext.TestName);
+            MsTestProperties.Add(LoggingPropertyNames.TestClassName, testContext.FullyQualifiedTestClassName);
+            MsTestProperties.Add(LoggingPropertyNames.TargetFramework, targetFramework.FrameworkName);
+            MsTestProperties.Add(LoggingPropertyNames.OsInfo, operatingSystem);
+        }
+
+        public void Trace(string message, SeverityLevel severity = SeverityLevel.Information, IDictionary<string, string> extraProperties = null)
```

Comment:
I am confused about how this method call works, as opposed to how `TestLogger.Trace` works